sdk/hybridaks/Azure.ResourceManager.HybridContainerService/src/Generated/ProvisionedClustersResponseResource.cs

```diff
@@ -0,0 +1,588 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+using Azure.ResourceManager.HybridContainerService.Models;
+using Azure.ResourceManager.Resources;
+
+namespace Azure.ResourceManager.HybridContainerService
+{
+    /// <summary>
+    /// A Class representing a ProvisionedClustersResponse along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct a <see cref="ProvisionedClustersResponseResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetProvisionedClustersResponseResource method.
+    /// Otherwise you can get one from its parent resource <see cref="ResourceGroupResource" /> using the GetProvisionedClustersResponse method.
+    /// </summary>
+    public partial class ProvisionedClustersResponseResource : ArmResource
```

Comment:
The generator takes the response model in swagger as the resource name, but we should get rid of the `Response` suffix as a resource anme. Rename `ProvisionedClustersResponse` to `ProvisionedCluster`. If there is an existing `ProvisionedCluster(s)`, rename that model to something else.

Reply:
Thanks for the suggestion, I've renamed it now